### PR TITLE
Enhancement/standardized naming

### LINKED ISSUE #1449: Rename the logger to log
Use more standardized naming in the log functionality.

```
import { logger } from '../../logger';
...
logger.info('Extracting classes');
```

```
import { log } from '../../logger';
...
log.info('Extracting classes');
```

### PR DESCRIPTION
**Pull request by harshilparmar**
## :bookmark_tabs: Summary
The codebase was using `logger` and `log` functionality interchangeably. So as discussed in #1449 I have changed that to `logger`. 
Resolves #1449

### :clipboard: Tasks
Make sure you
- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md) 
- [x] :bookmark: targeted `develop` branch 

Let me know if other changes are needed.

### CODE REVIEW

File: src/dagre-wrapper/clusters.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import intersectRect from './intersect/intersect-rect';
-import { logger as log } from '../logger'; // eslint-disable-line
+import { logger } from '../logger'; // eslint-disable-line
```
[the remark this answers is not in the document]
Comment (harshilparmar) [reply src/dagre-wrapper/clusters.js:2]: It's already there in the main repo. I haven't added that.

[the remark this answers is not in the document]
Comment (sidharthv96) [reply src/dagre-wrapper/clusters.js:2]: Yeah, but can you try if does removing that cause any issues with eslint? If not, you can remove it.

**Review by knsv — CHANGES_REQUESTED** [review]
Make a bigger change and rename all log instances from logger to log as per our discussion.

**Comment by harshilparmar** [comment]
@knsv Suggestion: What about changing logger.js to log.js !!

**Review by knsv — APPROVED** [review]
Perfect, thanks!

**Pull request merged**